crates/wdk-build/rust-driver-makefile.toml

```diff
@@ -303,6 +303,24 @@ args = [
 private = true
```

Comment:
https://github.com/microsoft/windows-drivers-rs/blob/10a8e3742b7834e816f6a2ce630b46533bcec357/crates/wdk-build/src/cargo_make.rs#L824

Previously, there was an assumption that there must be wdk-build in the workspace. This is no longer true now there can be a null driver in the workspace without any actual driver code being built (so no wdk-build in the graph).

In the wdk-build only case, we should also see if putting a workspace dep on wdk-build yields anything in the cargo metadata output.

maybe also worth putting a specific message for the case of no-wdk in workspace since that would typically be a null-driver-only workspace scenario